src/main/java/com/github/cameltooling/lsp/internal/completion/CamelComponentSchemesCompletionsFuture.java

```diff
@@ -79,7 +79,7 @@ public CamelComponentSchemesCompletionsFuture(CamelUriElementInstance uriElement
 				result.add(completionItem);
 			}
 		} catch (Exception ex) {
-			LOGGER.error("Error retrieving existing " + uriElement.getComponentName() + " endpoints!", ex);
+			LOGGER.error(String.format("Error retrieving existing %s endpoints!", uriElement.getComponentName()), ex);
```

Comment:
worth double-checking but `LOGGER.error("Error retrieving existing {} endpoints!", uriElement.getComponentName(), ex);` seems better

Reply:
tested that it is working.
provided PR https://github.com/camel-tooling/camel-language-server/pull/525